src/input.ts

```diff
@@ -21,6 +22,9 @@ export const parseInput = (env: Record<string, string | undefined>): Input => {
   const continueAfterSeconds = env["INPUT_CONTINUE-AFTER-SECONDS"]
     ? parseInt(env["INPUT_CONTINUE-AFTER-SECONDS"], 10)
     : undefined;
+  const sameBranchOnly =
+    env["INPUT_SAME-BRANCH-ONLY"] === "true" ||
+    env["INPUT_SAME-BRANCH-ONLY"] === undefined; // true if not specified
```

Comment:
Could we keep the default the same and make this an opt in? That would make this a non breaking change for people that depend on the existing behavior.

Reply:
Oh nevermind that _is_ the current behavior